Package.swift

```diff
@@ -5,10 +5,10 @@ import PackageDescription
 let package = Package(
     name: "SwiftPhoenixClient",
     platforms: [
-        .macOS(.v10_12),
-        .iOS(.v10),
-        .tvOS(.v10),
-        .watchOS(.v3)
+        .macOS(.v10_15),
+        .iOS(.v13),
+        .tvOS(.v13),
+        .watchOS(.v6)
```

Comment:
watchOS doesn't allow low level network connections, such as NSURLWebsocketTask, and I think it should just be removed.

https://developer.apple.com/documentation/technotes/tn3135-low-level-networking-on-watchos

Reply:
Happy to remove watchOS and update if you prefer. But perhaps worth leaving it in and adding a note to the docs or readme?

It should build fine, and if I read that TN right, there are potential use cases on watchOS, even if not the best idea. (You probably shouldn't stream audio via channel messages, but maybe handy as a control channel).

Reply:
Let's go ahead and remove it. In addition to these changes, the podspec should also be updated to reflect the new min versions

Reply:
OK, removed. I pulled some obsolete description from the pod spec while I was in there. Gotta get it updated before podspec repo goes read-only!

swift package dump-package succeeds.
I don't know how to validate a pod spec but these were pretty light changes.